packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -60,6 +63,17 @@ function processRequest(
   const type = request.type === RequestType.XHR ? ResourceType.XHR : ResourceType.FETCH
 
   const matchingTiming = matchRequestTiming(request)
+
+  if (!hasLoggedMissingResource && !matchingTiming) {
+    addTelemetryDebug('Missing PerformanceResourceTiming', {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏matchTiming can be undefined for several reasons, what do you want to monitor exactly?‏